src/huggingface_hub/serialization/_torch.py

```diff
@@ -357,7 +692,7 @@ def _get_unique_id(tensor: "torch.Tensor") -> Union[int, Tuple[Any, ...]]:
 
     try:
         # for torch 2.1 and above we can also handle tensor subclasses
-        from torch.utils._python_dispatch import is_traceable_wrapper_subclass
+        from torch.utils._python_dispatch import is_traceable_wrapper_subclass  # type: ignore[import]
```

Comment:
Are all of these `# type: ignore[import]` mandatory? I don't have any issues locally with 

```sh
# pip freeze | grep -e torch -e mypy
mypy==1.5.1
mypy-extensions==1.0.0
pytorch-lightning==2.0.2
timm @ git+https://github.com/rwightman/pytorch-image-models.git@25ffac6880c2e9c441c070ffeff75d392b98f632
torch==2.5.1
torchaudio==2.3.1
torchmetrics==0.11.4
torchvision==0.20.1
```

Reply:
I think my local linter config was somehow broken, it's fixed now